langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -77,7 +77,7 @@ type XmlType xml;
 # + return - iterator object
 # Each item is represented by an xml singleton.
 public isolated function iterator(xml<ItemType> x) returns object {
```

Comment:
Shall we add some tests in LangLibXMLTest for the direct invocation of `lang.xml:iterator()` too?

```ballerina
    xml x = xml `<!--first-->`;
    xml<'xml:Comment> seq = <xml<'xml:Comment>> x.concat(xml `<!--second-->`);

    object {
        public isolated function next() returns record {| 'xml:Comment value; |}?;
    } iter = seq.iterator();

    // tests based on `iter.next()`.
```